### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by BDesselle**
README.md file was updated with answers to the supplied questions.

### CODE REVIEW

**Comment by BDesselle** [comment]
MVP Completed

File: README.md [hunk 1]
```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
+**The DOM is the Document Object Model. It is used as a bridge or API between the browser and our code.**
+
 2. What is an event?
 
+**An event is something that happens within the browser such as a mouse click or key press. We can listen for it to trigger our own chain of events.**
+
 3. What is an event listener?
 
+**An event listener is a piece of code that waits for a specific event to happen, and fires a callback in response to the event.**
+
```
Comment (kyranmccann) [inline README.md]: perfect!

File: components/Carousel/Carousel.js [hunk 2]
```diff
@@ -1,14 +1,44 @@
...
+      this.imgs[this.currentIndex].style.display = "none";
+      if (this.currentIndex === 3) {
+        this.currentIndex = 0;
+      } else {
+        this.currentIndex += 1;
+      }
+      this.imgs[this.currentIndex].style.display = "block";
+    }
+  }
+
```
Comment (kyranmccann) [inline components/Carousel/Carousel.js]: Love the stretch on the stretch with the infinite loop

File: index.html [hunk 3]
```diff
@@ -1,219 +1,250 @@
 <!DOCTYPE html>
 <html>
+  <head>
+    <title>Lambda Times</title>
+    <link rel="stortcut icon" href="./favicon.ico" />
+    <link rel="stylesheet" type="text/css" href="./CSS/index.css" />
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.min.js"
```
Comment (kyranmccann) [inline index.html]: jquery!

File: components/Tabs/Tabs.js [hunk 4]
```diff
@@ -1,72 +1,58 @@
...
-
 - With your selection in place, now chain a .forEach() method onto the tabs variable to iterate over the DOM NodeList
-
 - In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
-
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+let tabs = document.querySelectorAll(".tab");
+tabs.forEach(tab => new TabLink(tab));
```
Comment (kyranmccann) [inline components/Tabs/Tabs.js]: great work here, brandon. and super easy to read!

**Pull request merged**